gcc/rust/checks/errors/borrowck/rust-bir-builder-internal.h

```diff
@@ -102,9 +101,7 @@ struct BuilderContext
   FreeRegions fn_free_regions{{}};
 
 public:
-  BuilderContext ()
-    : tyctx (*Resolver::TypeCheckContext::get ()),
-      resolver (*Resolver::Resolver::get ())
+  BuilderContext () : tyctx (*Resolver::TypeCheckContext::get ())
```

Comment:
...and just add it something like this:

```suggestion
  BuilderContext () : tyctx (*Resolver::TypeCheckContext::get ()), resolver(ImmutableNameResolutionCtx::get().resolver())
```
```